packages/nextjs/.prettierrc.js

```diff
@@ -6,4 +6,4 @@ module.exports = {
   "importOrder": ["^react$", "^next/(.*)$", "<THIRD_PARTY_MODULES>", "^@heroicons/(.*)$", "^~~/(.*)$"],
   "importOrderSortSpecifiers": true,
   "plugins": [require.resolve("@trivago/prettier-plugin-sort-imports")],
-}
+}
```

Comment:
Sorry to ask you this, but can you revert this change as well?
    
It’s very common for projects to have this last empty line in the file. This is important to follow the Unix pattern, avoid problems with some tools, and keep the code consistent.
    
I’m also asking you this because my duty as a reviewer is to protect the git change history, and this change would have no meaning to the package.json history.